src/compiler/emitter.ts

```diff
@@ -2709,6 +2709,17 @@ var __param = (this && this.__param) || function (paramIndex, decorator) {
                     }
                     else {
                         if (node.flags & NodeFlags.Default) {
+                            if (compilerOptions.module === ModuleKind.CommonJS || compilerOptions.module === ModuleKind.AMD || compilerOptions.module === ModuleKind.UMD) {
+                                write("Object.defineProperty(exports, \"__esModule\", {");

```

Comment:
`Object.defineProperty` is NOT available in ES3.
If compile target is ES3, you may not use it.
Just `exports.__esModule = true;` like babel's [loose mode](http://babeljs.io/docs/advanced/loose/#es6-modules).